cz.cvut.edu.comparch.qtrvsim.json

```diff
@@ -0,0 +1,36 @@
+{
+    "app-id" : "cz.cvut.edu.comparch.qtrvsim",
+    "runtime" : "org.kde.Platform",
+    "runtime-version" : "6.2",
+    "sdk" : "org.kde.Sdk",
+    "command" : "qtrvsim_gui",
+    "rename-desktop-file" : "qtrvsim.desktop",
+    "rename-icon" : "qtrvsim_gui",
+    "finish-args" : [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "modules" : [
+        {
+            "name" : "qtrvsim",
+            "buildsystem" : "cmake-ninja",
+            "builddir" : true,
+            "config-opts" : [
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://github.com/cvut/qtrvsim/archive/refs/tags/v0.9.3.tar.gz",
+		    "sha256": "9a1580aee3751ff396dd084f656b6b94291beaacc0860583c126ab3e5baed39e"
+                },
+                {
+                    "type": "patch",
+		    "path": "0001-Packaging-drop-older-releases-from-metainfo.patch"
+		}
```

Comment:
it would be nicer is the indentation of the .json file was consistent. (it's not a blocker)